### PR TITLE
Chores/test setup

### PR DESCRIPTION
**Pull request by Zsaschz**
Setup jest for unit testing

to run test please run `npm run test`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "babel-jest": "^27.5.1",
+    "jsdom-global": "^3.0.2"
+  },
+  "jest": {
+    "moduleFileExtensions": [
+      "js",
+      "json",
+      "vue"
+    ],
+    "testURL": "http://localhost/",
```
Comment (AldiNFitrah) [inline package.json:25]: with this value, is it runnable on github workflow?

File: tests/unit/example.spec.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { mount } from "@vue/test-utils";
+import Index from "../../src/resources/app/pages";
+
+// every test file should includes AT LEAST ONE "describe()", "it()", and "expect()"
```
Comment (AldiNFitrah) [inline tests/unit/example.spec.js:4]: `should include`

File: tests/unit/example.spec.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+describe("displays message", () => {
+  // mount() returns a wrapped Vue component we can interact with
+  it("should renders", () => {
+    // The component to test
```
Comment (AldiNFitrah) [inline tests/unit/example.spec.js:8]: "the component to be tested" or "the to-be-tested component"

File: package.json (hunk 1 above)
Comment (Zsaschz) [reply package.json:25]: well it runs on pipeline for now

File: tests/unit/example.spec.js (hunk 2 above)
Comment (Zsaschz) [reply tests/unit/example.spec.js]: fixed

File: tests/unit/example.spec.js (hunk 3 above)
Comment (Zsaschz) [reply tests/unit/example.spec.js]: fixed

**Review by AldiNFitrah — APPROVED** [review]
thanks a lot than, it will save so much of our time!

**Review by IlmaMannix — APPROVED** [review]
Thanks for your efforts. LGTM 👍😎

**Pull request merged**